docs/content_management/schedule_publishing.md

```diff
@@ -51,20 +52,20 @@ To browse all the future events, use the [Calendar widget](#calendar-widget).
 
 ## Date-based hiding
 
-When your content item is published, you can schedule a date and time and the content will be hidden at that time.
+When your content item is published, you can schedule a date and time and the content is hidden at that time.
```

Comment:
```suggestion
When your content item is published, you can schedule a date and time for the content to become hidden.
```
I feel like the current sentence still works better with "will be", I've tried rewriting this to avoid this word.